examples/classification/4-tutorials/plot_main-tutorial-classification.py

```diff
@@ -0,0 +1,288 @@
+"""
+===========================
+Tutorial for classification
+===========================
+
+
+In this tutorial, we compare the prediction sets estimated by the conformal
+methods implemented in MAPIE on a toy two-dimensional dataset.
+
+Throughout this tutorial, we will answer the following questions:
+
+- How does the number of classes in the prediction sets vary according to
+  the significance level ?
+
+- Is the chosen conformal method well calibrated ?
+
+- What are the pros and cons of the conformal methods included in MAPIE ?
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+from sklearn.model_selection import train_test_split
+from sklearn.naive_bayes import GaussianNB
+
+from mapie.classification import MapieClassifier
+from mapie.metrics import (
+    classification_coverage_score, classification_mean_width_score
+)
+
+##############################################################################
+# 1. Conformal Prediction method using the softmax score of the true label
+# ------------------------------------------------------------------------
+#
+# We will use MAPIE to estimate a prediction set of several classes such
+# that the probability that the true label of a new test point is included
+# in the prediction set is always higher than the target confidence level :
+# :math:`P(Y \in C) \geq 1 - \alpha`.
```

Comment:
`P(Y_{n+1} \in \hat{C}_{n, \alpha}(X_{n+1})`
To have the save notations as in Romano

Reply:
Done